packages/client/tsconfig.json

```diff
@@ -1,25 +1,45 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "baseUrl": "src",
+    "downlevelIteration": true,
     "lib": [
+      "esnext",
       "dom",
-      "dom.iterable",
-      "esnext"
+      "dom.iterable"
     ],
+    "module": "esnext",
+    "target": "esnext",
+    "sourceMap": true,
     "allowJs": true,
-    "skipLibCheck": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
     "isolatedModules": true,
+    "jsx": "react-jsx",
+    "moduleResolution": "node",
+    "forceConsistentCasingInFileNames": false,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "noUnusedParameters": true,
+    "noUnusedLocals": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "skipLibCheck": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "noFallthroughCasesInSwitch": true
   },
+  "exclude": [
+    "e2e",
+    "node_modules",
+    "build",
+    "scripts",
+    "webpack",
```

Comment:
Is stuff like `webpack` and `setupTests` relevant here?

Reply:
Yeah it's "just" an exclusion list but you are right, let's keep it clean. I kept only node modules and in case we add it later